app/views/claims/support/claims/_details.html.erb

```diff
@@ -85,3 +85,41 @@
     <% row.with_value(text: humanized_money_with_symbol(claim.amount)) %>
   <% end %>
 <% end %>
+
+  <% if claim.status == "clawback_requested" || claim.status == "clawback_in_progress" %>
```

Comment:
```suggestion
  <% if claim.clawback_requested? || claim.clawback_in_progress? %>
```

When we have enums we gain some useful helper methods for these sorts of checks! What do you think about updating the status logic with these methods?